core-chart/values.yaml

```diff
@@ -5,9 +5,10 @@
 
 # Container/chart image versions. These values are set during the chart release process,
 # please do not change them unless you know what you are doing.
-HELM_VERSION: "3.15.0"
-KUBECTL_VERSION: "1.30.1"
+HELM_VERSION: "3.16.1"
+KUBECTL_VERSION: "1.31.0"
 KUBESTELLAR_VERSION: "0.24.0"
+TRANSPORT_VERSION: "0.24.0"
```

Comment:
I am not a fan of adding self-references. OTOH, I _am_ a fan of making it clear how an out-of-tree transport-controller could be created and used. If this were a Makefile or script then we could say _here_ that `TRANSPORT_VERSION` defaults to whatever `KUBESTELLAR_VERSION` is --- but, as far as I know, you can not do such a thing in a Helm values file.

Reply:
No values in in `value.yaml` cannot reference other values in the same file.
We could compromise in the way that if `TRANSPORT_VERSION` is not set, the it defaults to `KUBESTELLAR_VERSION` by changing the PCH value substitution.

Reply:
Yes, but that requires duplicating the defaulting logic at every use rather than declaring it once at the definition. I am not asking for a change here.

Reply:
it's not complicated, I made the change in the new commit
the PCH will use the TRANSPORT_VERSION if defined and default to KUBESTELLAR_VERSION otherwise

Reply:
I see that there is only one use, besides the one in the unused file `core-charttemplatespostcreatehookswds.yaml`, so the complexity as measured in source tokens is about the same (currently).